src/api/public/apidocs-new/paths/staging_project_name_staging_projects_staging_project_name_copy_staging_project_copy_name.yaml

```
@@ -0,0 +1,43 @@
+post:
+  summary: Copy a staging project
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/staging_project_name.yaml'
+    - in: path
+      name: staging_project_copy_name
+      schema:
+        type: string
+      required: true
+      description: The name of the staging project's copy
+      example: "openSUSE:Factory:Staging:B"
+  responses:
+    '200':
+      description: Get the state of a staging project.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          example:
+            code: 'ok'
+            summary: 'ok'
```

Sorry, I didn't realize this before. The description doesn't look right but anyway this could be a simple reference:

```suggestion
    '200':
      $ref: '../components/responses/succeeded.yaml'
```